### PR TITLE
CANS-357 move age toggle

### PR DESCRIPTION
**Pull request by abelayt**
## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [X ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X ] My code follows the code style of this project.
- [X ] I ran all my tests locally which were green.
- [X ] My code adds no new issues to Code Climate
- [X ] I ran all the linters for the project.
- [X ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [X ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -70,4 +70,4 @@ services:
         source: ./spec
         target: /app/spec
     environment:
-      CANS_WEB_BASE_URL: ${CANS_WEB_BASE_URL}
+      CANS_WEB_BASE_URL: ${CANS_WEB_BASE_URL}
```
Comment (edgibbs) [inline docker-compose.yml]: Can probably just revert this change.

File: yarn.lock [hunk 2]
```diff
@@ -4,19 +4,19 @@
 
 "@babel/code-frame@7.0.0-beta.44":
   version "7.0.0-beta.44"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz#2a02643368de80916162be70865c97774f3adbd9"
+  resolved "http://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz#2a02643368de80916162be70865c97774f3adbd9"
```
Comment (edgibbs) [inline yarn.lock]: Should yarn lock change for just a css change?

File: docker-compose.yml (hunk 1 above)
Comment (abelayt) [reply docker-compose.yml]: Just did that. I needed to comment out some services for local build, that  was why and forgot the last line.

File: yarn.lock (hunk 2 above)
Comment (abelayt) [reply yarn.lock]: I remove yarn lock for a new local build of a new branch.
